### ISSUE TITLE
Gutenberg 16.7: unable to type certain letters in the editor canvas on Simple Sites

### ISSUE THREAD

**Issue by liviopv** [Automattic/wp-calypso#82431]
### Quick summary

With Gutenberg 16.7 running on edge, When typing on a Page or Template in the Site Editor, the editor is not registering certain letters ("n", "r", "s", "t"), as if the keypress was triggering a different editor, making the focus switch from the editor canvas.

https://github.com/Automattic/wp-calypso/assets/14153300/abfbf501-9e2f-4536-9db6-c9f407e56e87

p1695983653368489-slack-CBTN58FTJ

### Steps to reproduce

1. Make sure Gutenberg 16.7 is running
2. Open the site editor
3. Navigate to either a template or a page
4. Add a text block (Paragraph, Heading)
5. Try to type the letters "n", "r", "s", "t"

### What you expected to happen

It should print the letter.

### What actually happened

The editor seems to switch focus from the block being edited and it doesn't register the keypress

### Impact

One

### Available workarounds?

No and the platform is unusable

### Platform (Simple and/or Atomic)

Simple

**Comment by annezazu** [comment]
Noting that I can't replicate this in a self hosted site using 6.3.1 and GB 16.7.

**Comment by cuemarie** [comment]
Folks have triaged and ID'd a PR as a culprit here: p1696272235211389/1695983653.368489-slack-CBTN58FTJ

TLDR - It appears to a "WPCOM-specific (site) editor component" interacting poorly with a recent GB change; currently looking for the right person to take this on

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – **Replicated** 
- Tested on Atomic – Could Not Replicate
- Replicable outside of Dotcom – ~No~ - [See below](https://github.com/Automattic/wp-calypso/issues/82431#issuecomment-1745649060)

📌 FINDINGS/SCREENSHOTS/VIDEO
Replicated on a new Free (Edge) site running 16.7.0. 

My tests also show a conflict with the character `a` once the issue starts happening:

https://github.com/Automattic/wp-calypso/assets/27249804/e16fce1e-9b65-417b-9a73-ddc041ff9236

📌 ACTIONS
- Triaged

**Comment by annezazu** [comment]
I just replicated this on a self hosted site but with Jetpack installed. When I deactivated Jetpack, it works just fine. I'm using Jetpack Version 12.7-a.3. cc @kraftbj

**Comment by kraftbj** [comment]
@sdixon194 Want to get this in front of you too while you're holding down the fort while we're at the DM. I'm unsure how this may be related to the other Gutenberg impacting issues.

**Comment by anomiex** [comment]
This reminds me of 5214476-zd-woothemes. Try deactivating just Jetpack's "Notifications" module.

If that works, it's probably something in https://s0.wp.com/wp-content/mu-plugins/notes/admin-bar-v2.js. I have no idea who if anyone owns that.

**Comment by kraftbj** [comment]
~I could duplicate with 'n' but not the other letters. @anomiex's suggestion to try deactivating Notifications did resolve the issue, so it is likely a JS conflict with the notifications code.~

Edit: I caught a bug in the post editor, not this one. Disregard this comment.

**Comment by kraftbj** [comment]
For the separate issue with the post editor I mentioned above, it is temporarily resolved via https://github.com/Automattic/jetpack/pull/33458

**Comment by cometgrrl** [comment]
This has been resolved. See p1696624552891889/1695983653.368489-slack-CBTN58FTJ. Also, I retested with gutenberg-edge sticker enabled on my simple site and confirmed the fix.
